index.html

```diff
@@ -11,24 +11,27 @@
     <header>
       <div class="header-container">
         <div class="logo">
-          <a href="#" id="welcome-logo">Welcome</a>
+          <a href="#Home" id="welcome-logo">Welcome</a>
         </div>
         <nav class="mobile-menu-btn">
-          <button type="button" class="mobile-btn">
+          <button type="button" class="mobile-btn mobile-hamburger-btn">
             <img src="./icons/mobile-menu-btn.svg" alt="mobile menu" />
           </button>
         </nav>
         <nav class="desktop-nav">
+          <button type="button" class="mobile-btn mobile-close-btn">
+            <img src="./icons/Icon-Cancel.png" alt="mobile close button" />
+          </button>
           <ul class="animate">
-            <li><a href="#">Portoflio</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Contact</a></li>
+            <li><a href="#Portoflio">Portoflio</a></li>
+            <li><a href="#About">About</a></li>
+            <li><a href="#Contact">Contact</a></li>
```

Comment:
- [ ] You've done a great job implementing the mobile button! You're almost there! Kindly increase the font weight on your navigation links to better fit the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) you've chosen.
# Your design vs. Figma
## Yours
![yours](https://user-images.githubusercontent.com/69858269/167718932-76254d22-c5c6-41dd-a726-df4258736fef.PNG)
## Figma
![figma](https://user-images.githubusercontent.com/69858269/167718943-6b6246c1-3f96-4c40-a6b0-1ac4c771cb98.PNG)